submissions/vasylpaliy/html_css_popup/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="style.css">
+      <title>popup</title>
+   </head>
+   <body>
+      <header>
+         <nav>
```

Comment:
Your navigation depends on viewport height(avoid doing that) [watch video](https://share.vidyard.com/watch/44b9xRgLsFa7CiGcBntL4K?) please fix it.
Also fix the aspect ratio and alignment of icons. [Read about aspect ratio](https://jakearchibald.com/2022/img-aspect-ratio/)

Reply:
- [x] Fixed the part of navigation, so it no longer depends on viewport height . Also, done the same with icons from popup menu.
- [x] Fixed the alighment of icons. Now they are centred vertically and horizontally. 
- [ ]  Aspect ratio. I tried to use it with my popup containers to prevent them from jumping, but it did not work. Wasted on it 2 days 🔥

Reply:
[Read this article](https://jakearchibald.com/2022/img-aspect-ratio/) to find out how to deal with an image aspect ratio